### ISSUE TITLE
Static title across slides

### ISSUE THREAD

**Issue by Hugo-Trentesaux** [hakimel/reveal.js#2275]
I am trying to do a fixed title for several consecutive slides i.e:

### Title

Slide 1

---

### Title

Slide 2

---

### Title

Slide 3

---

with a transition for the slide content but not for the title, because a transition lets think I move to something else while actually I only continue the content on an other slide.

I thought it could be achieved with fragments, but it appears not. Is there a parameter to tune this ?

**Comment by cMadan** [comment]
If you're okay with just using fragments, this can be done as a single slide, having div's that show/hide in a sequence defined with data-fragment-index.

**Comment by Hugo-Trentesaux** [comment]
Yes, but it seems not to be possible to have the "slide" transition effect (https://revealjs.com/#/transitions) on a fragment. And I do not know if it is easy to achieve this.

**Comment by Yanbo** [comment]
I also met this issues, is there any native way to achieve the static title in consecutive slide?

**Comment by cyclux** [comment]
If anyone stumbles on this:
Just put something like this above the <section> tags
```html
...
<body>
<div class="reveal">
	<h3 style="text-align: center;">Your static title</h3>
        <section>...<\section>
        ....
<\div>
<\body>
...
```
